gb-cpu/src/microcode/write.rs

```diff
@@ -95,7 +95,8 @@ pub fn hram(ctl: &mut MicrocodeController, state: &mut State) -> MicrocodeFlow {
 /// cycle
 pub fn ind16(ctl: &mut MicrocodeController, state: &mut State) -> MicrocodeFlow {
     let addr = ctl.pop_u16();
+    let value = ctl.pop();
     state.write_bus(addr, ctl.pop());
-    state.write_bus(addr + 1, ctl.pop());
+    state.write_bus(addr + 1, value);
```

Comment:
Would be clearer if values were stored in variables with a name adapted to their functions. Ex: `lower` and `upper`